<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestUIClass</class>
 <widget class="QMainWindow" name="TestUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="textOrder">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
    <property name="plainText">
     <string>*IDN?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSendOrder">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>order</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editSrcName">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TCPIP0::192.168.10.161::inst0::INSTR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>order</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textOrderShow">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>221</width>
      <height>231</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TestUI.qrc"/>
 </resources>
 <connections/>
</ui>
